### ISSUE TITLE
Figure out why we can't helm upgrade the production server

### ISSUE THREAD

**Issue by moorepants** [LibreTexts/metalc#75]
This is the current thing to try:

Unable to upgrade still but suspect the problem is with chick4. We plan to shut it down.

**Comment by celine168** [comment]
Just upgraded now. We ran `kubectl delete node chick4`, which removes chick4 from the list of nodes in the Kubernetes API. This seemed to allow JupyterHub to upgrade. 

For documenting this issue: we disabled scheduling on chick4 because its hard disks were spinning out of control. It included pods such as `continuous-image-puller-xxxx` and `hook-image-puller-xxxx` that were always in a `Pending` state. Deleting these pods manually would cause them to restart on their own almost instantaneously. 

I believe `hook-image-puller` pods pull the Docker images from DockerHub and they must be complete for JupyterHub to complete the upgrade. Once the upgrade is complete, the `hook-image-puller` pods disappear.
